docs_source_files/content/introduction/about_this_documentation.es.md

```diff
@@ -4,32 +4,16 @@ weight: 4
 ---
 
 {{% notice info %}}
```

Comment:
@AlexAndradeSan could you please remove the “notice”?
The page has been fully translated, so it’s not needed anymore.